samples/17.multilingual-bot/src/main/java/com/microsoft/bot/sample/multilingual/MultiLingualBot.java

```diff
@@ -1,2 +1,133 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
-// Licensed under the MIT License.
\ No newline at end of file
+// Licensed under the MIT License.
+package com.microsoft.bot.sample.multilingual;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.microsoft.bot.builder.ActivityHandler;
+import com.microsoft.bot.builder.UserState;
+import com.microsoft.bot.builder.StatePropertyAccessor;
+import com.microsoft.bot.builder.TurnContext;
+import com.microsoft.bot.builder.MessageFactory;
+import com.microsoft.bot.schema.ChannelAccount;
+import com.microsoft.bot.schema.Activity;
+import com.microsoft.bot.schema.CardAction;
+import com.microsoft.bot.schema.ActionTypes;
+import com.microsoft.bot.schema.SuggestedActions;
+import com.microsoft.bot.schema.Attachment;
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.concurrent.CompletableFuture;
+
+/**
+ * This bot demonstrates how to use Microsoft Translator.
+ * More information can be found
+ * here https://docs.microsoft.com/en-us/azure/cognitive-services/translator/translator-info-overview.
+ */
+public class MultiLingualBot extends ActivityHandler {
+    private static final String WELCOME_TEXT = "This bot will introduce you to translation middleware. "
+        + "Say 'hi' to get started.";
+    private static final String ENGLISH_ENGLISH = "en";
+    private static final String ENGLISH_SPANISH = "es";
+    private static final String SPANISH_ENGLISH = "in";
+    private static final String SPANISH_SPANISH = "it";
+
+    private UserState userState;
+    private StatePropertyAccessor<String> languagePreference;
+
+    public MultiLingualBot(UserState withUserState) {
+        if (withUserState != null) {
+            userState = withUserState;
+        } else {
+            throw new NullPointerException("userState");
+        }
+        languagePreference = userState.createProperty("LanguagePreference");
+    }
+
+    @Override
+    protected CompletableFuture<Void> onMembersAdded(List<ChannelAccount> membersAdded,
```

Comment:
Checkstyle warns about missing javadoc for overriden methods

Reply:
Yes, but there isn't a javadoc for this, what should i do?